app/app.go

```diff
@@ -407,32 +531,54 @@ func NewGaiaApp(
 	app.MountTransientStores(tkeys)
 	app.MountMemoryStores(memKeys)
 
-	// initialize BaseApp
+	anteHandler, err := NewAnteHandler(
+		HandlerOptions{
+			HandlerOptions: ante.HandlerOptions{
+				AccountKeeper:   app.AccountKeeper,
+				BankKeeper:      app.BankKeeper,
+				FeegrantKeeper:  app.FeeGrantKeeper,
+				SignModeHandler: encodingConfig.TxConfig.SignModeHandler(),
+				SigGasConsumer:  ante.DefaultSigVerificationGasConsumer,
+			},
+			IBCChannelkeeper: app.IBCKeeper.ChannelKeeper,
+		},
+	)
+	if err != nil {
+		panic(fmt.Errorf("failed to create AnteHandler: %s", err))
+	}
+
+	app.SetAnteHandler(anteHandler)
 	app.SetInitChainer(app.InitChainer)
 	app.SetBeginBlocker(app.BeginBlocker)
-	app.SetAnteHandler(
-		ante.NewAnteHandler(
-			app.AccountKeeper, app.BankKeeper, ante.DefaultSigVerificationGasConsumer,
-			encodingConfig.TxConfig.SignModeHandler(),
-		),
-	)
 	app.SetEndBlocker(app.EndBlocker)
-	app.UpgradeKeeper.SetUpgradeHandler("Gravity-DEX",
-		func(ctx sdk.Context, plan upgradetypes.Plan) {
-			var genState liquiditytypes.GenesisState
-			genState.Params = liquiditytypes.DefaultParams()
-			genState.Params.PoolCreationFee = sdk.NewCoins(sdk.NewCoin("uatom", sdk.NewInt(40000000)))
-			app.LiquidityKeeper.InitGenesis(ctx, genState)
-		})
+
+	app.UpgradeKeeper.SetUpgradeHandler(
+		upgradeName,
+		func(ctx sdk.Context, _ upgradetypes.Plan, _ module.VersionMap) (module.VersionMap, error) {
+			// TODO: Verify these parameters are correct and intended.
+			app.IBCKeeper.ConnectionKeeper.SetParams(ctx, ibcconnectiontypes.DefaultParams())
+
+			fromVM := make(map[string]uint64)
+			for moduleName := range app.mm.Modules {
+				fromVM[moduleName] = 1
```

Comment:
(didn't get an answer for https://github.com/cosmos/gaia/pull/942#discussion_r709064892) Did the hub ever integrate the vesting tracking delegation issue? (https://github.com/cosmos/cosmos-sdk/pull/8865)? If yes, we would also need `fromVM[moduleName] = 2`

Reply:
AFAIK the vesting issue still exists on the Hub and was never fixed because that fix is included in 0.44.x. Is that correct @zmanian?

Reply:
OK, in this case we should revert 3f7c39babbac08788a43fd5b731bd46c2d39c8eb

Reply:
@amaurym we have the following as of right now:

```go
			fromVM := make(map[string]uint64)
			for moduleName := range app.mm.Modules {
				fromVM[moduleName] = 1
			}

			// override versions for _new_ modules as to not skip InitGenesis
			fromVM[authz.ModuleName] = 0
			fromVM[feegrant.ModuleName] = 0
```

Do we need to explicitly set the version of `auth`?

Reply:
> @amaurym we have the following as of right now:

Looking at the files, I see: https://github.com/cosmos/gaia/blob/f42f5023af99367d2bff60edd2a9e40ad9396df5/app/app.go#L568-L569

which we need to revert (commit 3f7c39b)

Reply:
@yaruwangway has done this